src/libraries/System.Security.Cryptography/tests/Interop.EVP.Kem.ExtraHandle.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Security.Cryptography;
+
+internal static partial class Interop
+{
+    internal static partial class Crypto
+    {
+        private static partial IntPtr GetExtraHandle(SafeEvpPKeyHandle handle)
+        {
+            _ = handle;
+            return IntPtr.Zero;
```

Comment:
When we get around to adding tests that open from a provider, and this isn't nullptr, it'll probably just need to do reflection to get the right answer.  But I'm cool with "yeah, I'll do that when I need to".

Reply:
>  it'll probably just need to do reflection to get the right answer.

I still don't think we need to do anything. Because adding tests from a provider should go through `SafeEvpPKeyHandle.OpenKeyFromProvider`, which is in `System.Security.Cryptography`, which will use the "real" partial.

Reply:
Ooooh, right.